### PR TITLE
tgif 4.2.5 (new formula)

### PR DESCRIPTION
**Pull request by fabianschuiki**
Add the tgif 4.2.5 2D drawing tool. Dates back quite a bit, but is still
being maintained. Very useful for drawing block diagrams.

### CODE REVIEW

File: tgif.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  depends_on :x11
+
+  def install
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.obj").write <<-EOS.undent
+    %TGIF 4.2.5
```
Comment (DomT4) [inline tgif.rb]: Can you indent this two spaces after the opening bracket above? Makes it easier to distinguish between main formula and file you're writing.

File: tgif.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    objshadow_info('#c0c0c0',2,2).
+    rotate_pivot(0,0,0,0).
+    spline_tightness(1).
+    page(1,"",1,'').
+    box('black','',64,64,128,128,0,1,1,0,0,0,0,0,0,'1',0,[
+    ]).
+
+    EOS
+    system "#{bin}/tgif", "-print", "-text", "-quiet", "test.obj"
+    system "test", "-e", "test.txt"
```
Comment (DomT4) [inline tgif.rb]: You can do this in Ruby. `assert File.exist?("test.txt")`

**Comment by fabianschuiki** [comment]
Done, and done.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/b3109bea795d50e2dac8cabe45a7da3822a91cda. Thank you for your contribution to Homebrew/x11, We appreciate it! :smiley_cat:

**Pull request closed without merging**